kratos/geometries/line_2d_2.h

```diff
@@ -693,8 +693,8 @@ class Line2D2 : public Geometry<TPointType>
 
     /**
      * @brief This method gives you all edges of this geometry.
-     * @details This method will gives you all the edges with one dimension less than this geometry.
-     * For example a triangle would return three lines as its edges or a tetrahedral would return four triangle as its edges but won't return its six edge lines by this method.
+     * @details This method will give you all the edges with one dimension less than this geometry.
+     * For example a triangle would return three lines as its edges or a tetrahedral would return four triangles as its edges but won't return its six edge lines by this method.
```

Comment:
```suggestion
     * For example a triangle would return three lines as its edges or a tetrahedron would return four triangles as its edges but won't return its six edge lines by this method.
```

Feel free to completely rewrite the docs here because *"a tetrahedral would return four triangles as its edges"* for example is **very** confusing to me.

Reply:
I have attempted to rewrite the documentation. Let me know how you feel about it. Thanks.

Reply:
Looks good, thanks!